.github/pull_request_template.md

```diff
@@ -1,8 +1,10 @@
-## Describe your changes
+<!-- Describe your changes --> 
 
-## Issue ticket number and link
+<!-- Issue ticket number and link (if applicable) -->
+
+<!-- Testing -->
+- [ ] I have tested those changes 
+- [ ] I have added tests (skip if just adding/editing providers)
+
+<!-- Testing instructions (if applicable) -->
 
-## Checklist before requesting a review (skip if just adding/editing APIs & templates)
-- [ ] I added tests, otherwise the reason is: 
-- [ ] I added observability, otherwise the reason is:
-- [ ] I added analytics, otherwise the reason is: 
```

Comment:
lgtm, usually I go simpler. Checkboxes are annoying and honestly, after a while, you stop caring.

```md
# 🤓 Context

# 🧪 How to tests?
```

Reply:
I would go for that if it was a private repo but since it is public I would be slightly more verbose